react/features/e2ee/middleware.ts

```diff
@@ -254,6 +270,30 @@ StateListenerRegistry.register(
         if (previousConference) {
             dispatch(toggleE2EE(false));
         }
+
+        conference.on(JitsiConferenceEvents.E2EE_VERIFICATION_AVAILABLE, pId => {
+            dispatch(participantUpdated({
+                e2eeVericationAvailable: true,
+                id: pId,
+                local: true
+            }));
+        });
+
+        conference.on(JitsiConferenceEvents.E2EE_VERIFICATION_READY, (pId, sas) => {
+            dispatch(openDialog(ParticipantVerificationDialog, { pId,
```

Comment:
Perhaps we should open the dialog and have the verify button disabled until ready instead? The process should be fast though...